### ISSUE TITLE
How to get WER from fine-tuning Wav2Vec2Seq2Seq

### ISSUE THREAD

**Issue by jubick1337** [pytorch/fairseq#3759]
#### What is your question?
How to get WER from fine-tuning Wav2Vec2Seq2Seq?
#### Code

fairseq-hydra-train task.data=/root/data model.w2v_path=/root/checkpoint.pt +task.autoregressive=true --config-dir /root/fairseq/examples/wav2vec/config/finetuning --config-name s2s_config

Config:
# @package _group_

common:
  fp16: true
  log_format: json
  log_interval: 200

checkpoint:
  no_epoch_checkpoints: true
  best_checkpoint_metric: loss

task:
  _name: audio_finetuning
  data: ???
  normalize: true
  labels: ltr

dataset:
  num_workers: 36
  max_tokens: 3200000
  skip_invalid_size_inputs_valid_test: true
  valid_subset: valid

distributed_training:
  ddp_backend: legacy_ddp
  distributed_world_size: 2

criterion:
  _name: cross_entropy
  zero_infinity: true

optimization:
  max_update: 80000
  lr: [0.00003]
  sentence_avg: true
  update_freq: [4]

optimizer:
  _name: adam
  adam_betas: (0.9,0.98)
  adam_eps: 1e-08

lr_scheduler:
  _name: tri_stage
  phase_ratio: [0.1, 0.4, 0.5]
  final_lr_scale: 0.05

model:
  _name: wav2vec_seq2seq
  w2v_path: ???
  apply_mask: true
  mask_prob: 0.65
  mask_channel_prob: 0.5
  mask_channel_length: 64
  layerdrop: 0.1
  activation_dropout: 0.1
  feature_grad_mult: 0.0
  freeze_finetune_updates: 0

#### What have you tried?
setting best_checkopoint_metric to were but it crashes
#### What's your environment?

 - fairseq Version (e.g., 1.0 or master): master
 - PyTorch Version (e.g., 1.0) 1.10.0
 - OS (e.g., Linux): Ubuntu 20.04 
 - How you installed fairseq (`pip`, source): source
 - Build command you used (if compiling from source):
 - Python version: 3.9
 - CUDA/cuDNN version: 11.4
 - GPU models and configuration: 2x RTX 3090
 - Any other relevant information:

**Comment by jubick1337** [comment]
add eval_wer: true to config
